src/js/actions.js

```diff
@@ -143,12 +143,12 @@ export const deleteSubMenu = (appID, menuID) => {
     }
 }
 
-export const subscribeButton = (appID, buttonName, isSubscribed) => {
+export const subscribeButton = (appID, buttonName) => {
     return {
         type: Actions.SUBSCRIBE_BUTTON,
         appID: appID,
         buttonName: buttonName,
-        isSubscribed: isSubscribed
+        isSubscribed: true
```

Comment:
@LiliiaShlikhtLuxoft why is it changed to `true` by default ?

Reply:
@ValeriiMalkov previously we got "isSibsribed" from Core, now we don't get it from Core. I use it only in case, when HMI works and subscribed, so it is used just to not rewrite complicating connections.